check.go

```diff
@@ -21,7 +21,7 @@ type Check struct {
 	Id          graphql.ID   `graphql:"id"`
 	Level       Level        `graphql:"level"`
 	Name        string       `graphql:"name"`
-	Notes       string       `graphql:"notes"`
+	Notes       *string      `graphql:"notes"`
```

Comment:
This is not needed as this is on the "response" side.  The passing of null is only needed on the "request variables" side IE input